portfolio.html

```diff
@@ -334,6 +334,7 @@ <h3>LET'S CONNECT</h3>
           Get It Touch
         </button>
       </form>
+      <div class="bottomline"></div>
     </section>
```

Comment:
Kindly try to match the background image with the Figma design in the contact me section
| Your design | Figma design |
| --- | --- |
| ![screencapture-raw-githack-amrendrakind-MyPortfolio-MV-feature-portfolio-html-2022-02-22-20_02_21](https://user-images.githubusercontent.com/24703171/155192191-d4a1f62e-3e6b-4488-9eb9-8b397feb34b1.png) | ![Template 2-Mobile-Main (1)](https://user-images.githubusercontent.com/24703171/155191878-09617953-e125-408a-aac5-5a92665cd6b8.png) |

the bottom shape does not appear.